example/cumulus-tf/cnm_response_task.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_lambda_function" "cnm_response_task" {
   function_name = "${var.prefix}-CnmResponse"
   s3_bucket     = "cumulus-data-shared"
-  s3_key        = "daacs/podaac/cnmResponse-1.2.1.zip"
+  s3_key        = "daacs/podaac/cnmResponse-1.0.6.zip"
```

Comment:
so we're going to an earlier version?

Reply:
I'm not sure where the version numbers for these packages were coming from before, but they don't match what is in the actual repo source code for these tasks, which is confusing. `1.0.6` matches the updated version number from the latest in the repo

Reply:
Sounds good, figured I'd check just in case :)

Reply:
Thanks for making that update!